scripts/test/globalSetup.ts

```diff
@@ -1,35 +1,31 @@
 import fs from 'node:fs/promises'
+import util from 'node:util'
 import url from 'node:url'
 import path from 'node:path'
 
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
 const nodeModulesPath = path.join(__dirname, '..', '..', 'packages', 'webdriver', 'node_modules')
 const gotPath = path.join(nodeModulesPath, 'got')
 const tmpGotPath = path.join(nodeModulesPath, 'tmp_got')
+const ERROR_MESSAGE = 'Renaming "got" dependency failed!\n' +
+'WebdriverIO needs to hide the "got" dependency (at "packages/webdriver/node_modules/got")\n' +
+'during the test to force Vitest to use our mocked version. WebdriverIO does this by\n' +
+'renaming "packages/webdriver/node_modules/got" to "packages/webdriver/node_modules/tmp_got"\n' +
+'during test setup and back again during test tear-down. \n\n' +
+'%s has failed.  Maybe because you are already running unit tests in a different\n' +
+'terminal.\n\n'
```

Comment:
```suggestion
const ERROR_MESSAGE = `Renaming "got" dependency failed!
'WebdriverIO needs to hide the "got" dependency (at "packages/webdriver/node_modules/got")
'during the test to force Vitest to use our mocked version. WebdriverIO does this by
'renaming "packages/webdriver/node_modules/got" to "packages/webdriver/node_modules/tmp_got"
'during test setup and back again during test tear-down.

'%s has failed.  Maybe because you are already running unit tests in a different
'terminal.
`
```